dataRetriever/factory/resolverscontainer/sovereignShardResolversContainerFactory.go

```diff
@@ -27,20 +27,58 @@ func NewSovereignShardResolversContainerFactory(shardContainer *shardResolversCo
 
 // Create returns a resolver container that will hold all resolvers in the system
 func (srcf *sovereignShardResolversContainerFactory) Create() (dataRetriever.ResolversContainer, error) {
-	_, err := srcf.shardResolversContainerFactory.Create()
+	err := srcf.generateTxResolvers(
+		factory.TransactionTopic,
+		dataRetriever.TransactionUnit,
+		srcf.dataPools.Transactions(),
+	)
 	if err != nil {
 		return nil, err
 	}
 
-	err = srcf.generateSovereignHeaderResolvers()
+	err = srcf.generateTxResolvers(
+		factory.UnsignedTransactionTopic,
+		dataRetriever.UnsignedTransactionUnit,
+		srcf.dataPools.UnsignedTransactions(),
+	)
+	if err != nil {
+		return nil, err
+	}
+
```

Comment:
Do you need to add `generateRewardResolver` here? Same as meta/shard resolvers container factory?

Reply:
No, no need to add it.
Every node will create and process its own rewards at EOE